src/ScanItems.test.js

```diff
@@ -925,4 +937,56 @@ describe('ScanItems', () => {
       });
     });
   });
+
+  describe('Detect item with use-at-location loan', () => {
+    it('should recognise use-at-location loans', async () => {
+      const props = {
+        stripes: {},
+        parentResources: {},
+        shouldSubmitAutomatically: true,
+        mutator: {
+          loans: {
+            reset: jest.fn(),
+            GET: (arg) => {
+              const re = /^itemId==(\d+) and status.name<>Closed$/;
+              const match = arg.params.query.match(re);
+              expect(match).not.toBeNull();
+              const id = match[1];
+
+              if (id === '123') {
+                return {
+                  totalRecords: 0,
+                  loans: [],
+                };
+              } else if (id === '456') {
+                return {
+                  totalRecords: 1,
+                  loans: [{ forUseAtLocation: { status: 'In use' } }],
+                };
+              } else /* id === '789' */ {
```

Comment:
Could you please clarify can we remove `/* id === '789' */`?

Reply:
Obviously I put it there for a reason. The test invokes this function with three different IDs of which this is the third. I would have just used `else if (id === '789')` but of course ESLint doesn't like that because it doesn't understand that those three cases exhaust the possibilities.